force-app/main/default/classes/HH_Container_LCTRL.cls

```diff
@@ -125,8 +133,10 @@ public with sharing class HH_Container_LCTRL {
         if (isHousehold) {
             householdFieldsToEdit.addAll(new Set<String>{
                 'BillingStreet', 'BillingCity', 'BillingState', 'BillingPostalCode',
-                 String.valueOf(Account.Undeliverable_Address__c), 'BillingCountry',
-                'BillingLatitude', 'BillingLongitude'});
+                'BillingCountry', 'BillingLatitude', 'BillingLongitude'});
+            if (canReadAccountUndeliverable) {
+                householdFieldsToEdit.add(String.valueOf(Account.Undeliverable_Address__c));
```

Comment:
won't this fail in a namespaced org?

Reply:
It shouldn't, we're referencing the Object.Fieldname directly instead of storing it as a String.